.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.6"
+  - "3.7"
+  # Tensorflow 1.x does not support python 3.8+
```

Comment:
We should probably update the readme with this. Maybe a line at the top somewhere of python versions we support: 3.6/3.7

Reply:
adding a badge, it will say 3 for now, but when we release the next version it will update to 3.6/3.7